script/slowest_pages

```diff
@@ -33,15 +33,15 @@ File.open(FILE).readlines.each do |line|
   path = "/#{controller}/#{action}"
   rec = data[path] ||= [0, 0, 0, 0]
   rec[0] += time
-  rec[1] += time * time
+  rec[1] += time**2
   rec[2] += 1
   rec[3] += 1 if robot
 end
 
 for path, rec in data
   sum, ssq, num = *rec
   rec[0] = avg = sum / num
-  rec[1] = sig = Math.sqrt(ssq / num - avg * avg)
+  rec[1] = Math.sqrt(ssq / num - avg**2)
```

Comment:
Fixes BinaryOperatorWithIdenticalOperands offenses
Fixes one UselessAssignment to Variable offense.

Reply:
This script is never really used, so no big deal, but avg**2 does a complicated power function, while avg * avg is just a plain old multiplication.  They are definitely not the same!  And I should point out that they don't even give the identical results typically.  avg * avg is more accurate.  avg**2 isn't even as easy to read.